craco.config.js

```diff
@@ -6,15 +6,21 @@ const path = require('path')
 module.exports = function() {
   return {
     webpack: {
-      alias: {},
+      alias: {
+        constants: path.resolve(__dirname, 'src/custom/constants')
```

Comment:
I assume u do this cause "constants" is an internal thingy for node? 
For their PR i changed their imports into "constants/" or "constants/index" (i cannot remember), and with that, it works the normal override

Reply:
Also, since we are adding alias, maybe we can simplify our imports in out custom components by adding an alias for their "src" like a @ or something

This way we don't need to user relative paths

Reply:
> For their PR i changed their imports into "constants/" or "constants/index" (i cannot remember), and with that, it works the normal override

doesn't work with craco

Reply:
> Also, since we are adding alias, maybe we can simplify our imports in out custom components by adding an alias for their "src" like a @ or something

I'm all for aliases

Reply:
> doesn't work with craco

weird, I think I tried that and it worked for me, we can talk on monday